source/data_structures.rst

```diff
@@ -1,108 +1,124 @@
 Data Structures
 ***************
 
-Data structures contain other objects. We will look at two: `lists`, `dictionaries`.
-
-Typical methods defined on data structures are ones that allow access and
-update items within it.
-
-As always first we explore how to create objects using literals and
-constructors, we then examine some methods typical of each object.
-
-Second, we often want to do something for each item in a data structure. This
-involves 'iterating' over it. We do this using the for loop.
+Data structures are objects that contain or organise other objects in a useful
+way. We will look at two datastructures in this lesson: lists and dictionaries
+("dicts").
 
+We will learn how to create these data structures and how to access their
+contents. We'll also look at how to do something with every item in a data
+structure. To do this we use loops.
 
 Lists
 =====
 
-A list object contains ordered items.
+A list is a data structure which contains objects in a specific order.
 
-Creation
---------
+Creating lists
+--------------
 
-.. code-block:: python
+To create a list from some other objects use square brackets like so::
 
-    >>> ['John', 'Eric', 'Michael', 'Terry']          # literal
-    >>> list('abc')                                   # `list` transform a string
-    ['a', 'b', 'c'] 
-    >>> type([1, 2, 3])
-    <class 'list'>
+    >>> monty_python = ['John', 'Terry G', 'Eric', 'Michael', 'Terry J', 'Graham']
```

Comment:
Did we learn variables already? I like this better but just checking this is a later section (pretty sure it is). Maybe explicitly state we're assigning to a variable.